### PR TITLE
Adding mission & vision, renaming training to handbook

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

File: source/process/handbook.rst [hunk 1]
```diff
@@ -1,3 +1,35 @@
+Team Handbook
+#############
+
+Mission & Vision
+-------
+
+Our mission is: "to give every organization the power to achieve their highest priorities through modern communication."
+
+We intend to disrupt the global workplace communications market with open core solutions delivering irreistable value.
```
Comment (lindy65) [inline source/process/handbook.rst]: irreistable > irresistible?

File: source/process/handbook.rst [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+
+Values are deliberate choices defining our behavior. When facing complexity, uncertainty, or ambiguity (CUA) we determine our point of view and our actions through the lens of our values: 
+
+Customer Obsession 
+==================
+We exist to serve customers. In everything we do, start with customer perspective and work backwards. Earn and keep their trust. 
+
+Ownership 
+======
+Own the outcomes of your activity. Don't drop the ball. When we see a vaccum on something important, we jump in--we never say "it's not my job."
```
Comment (lindy65) [inline source/process/handbook.rst]: vaccum > vacuum?

File: source/process/handbook.rst [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+======
+Own the outcomes of your activity. Don't drop the ball. When we see a vacuum on something important, we jump in--we never say "it's not my job." 
+
+High Impact 
+===========
+Figure out what matters most, and focus on those priorities. When deciding to work on low impact and high impact projects, choose high impact. 
+
+Be proud of what you build
+==========================
+Insist on high standards. Ensure that you're building highest quality products, services, and processes possible, and constantly raise the bar.
```
Comment (esethna) [inline source/process/handbook.rst]: "highest quality" -> "the highest quality"

Comment (esethna) [inline source/process/handbook.rst]: Propose "raise" -> "raising" to match the "ing" form of "building"?

File: source/process/handbook.rst [hunk 4]
```diff
@@ -1,3 +1,35 @@
...
+==================
+We exist to serve customers. In everything we do, start with customer perspective and work backwards. Earn and keep their trust. 
+
+Ownership 
+======
+Own the outcomes of your activity. Don't drop the ball. When we see a vacuum on something important, we jump in--we never say "it's not my job." 
+
+High Impact 
+===========
+Figure out what matters most, and focus on those priorities. When deciding to work on low impact and high impact projects, choose high impact.
```
Comment (esethna) [inline source/process/handbook.rst]: "low impact and high impact projects" -> "low impact or high impact projects"

**Comment by esethna** [comment]
@it33 added some minor proposed tweaks above^

**Comment by it33** [comment]
Thanks @esethna, updated

**Comment by esethna** [comment]
Thanks +1

File: source/process/handbook.rst [hunk 5]
```diff
@@ -1,3 +1,35 @@
...
+We intend to disrupt the global workplace communications market with open core solutions delivering irresistible value.
+
+Values
+--------
+
+Values are deliberate choices defining our behavior. When facing complexity, uncertainty, or ambiguity (CUA) we determine our point of view and our actions through the lens of our values: 
+
+Customer Obsession 
+==================
+We exist to serve customers. In everything we do, start with customer perspective and work backwards. Earn and keep their trust.
```
Comment (lfbrock) [inline source/process/handbook.rst]: Not sure how I feel about the wording "We exist to serve customers", but maybe we can discuss and update it later.

**Comment by it33** [comment]
Updated based on @lfbrock feedback

**Comment by jasonblais** [comment]
Thanks, looks good, +1

**Pull request merged**